src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/infer_request/config.cpp

```diff
@@ -1,28 +0,0 @@
-// Copyright (C) 2018-2023 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#include "behavior/infer_request/config.hpp"
-
-using namespace BehaviorTestsDefinitions;
-namespace {
-    const std::vector<std::map<std::string, std::string>> configs = {
-            {}
-    };
-
-    const std::vector<std::map<std::string, std::string>> InConfigs = {
-            {},
-            {{InferenceEngine::PluginConfigParams::KEY_CPU_THROUGHPUT_STREAMS, InferenceEngine::PluginConfigParams::CPU_THROUGHPUT_AUTO}},
-            {{InferenceEngine::PluginConfigParams::KEY_CPU_THROUGHPUT_STREAMS, InferenceEngine::PluginConfigParams::CPU_THROUGHPUT_NUMA}},
-            {{InferenceEngine::PluginConfigParams::KEY_CPU_THROUGHPUT_STREAMS, "8"}},
-            {{InferenceEngine::PluginConfigParams::KEY_CPU_BIND_THREAD, InferenceEngine::PluginConfigParams::NO}},
-            {{InferenceEngine::PluginConfigParams::KEY_CPU_BIND_THREAD, InferenceEngine::PluginConfigParams::YES}}
-    };
```

Comment:
What is an equivalent for the test, and for `CPU_BIND_THREAD` properties?

Reply:
I also just found that `InferRequestConfigTest` are migrated to separate `properties_tests.*pp`, so all is clear for me now. Thanks!